upi/openstack/network.yaml

```diff
@@ -28,6 +28,18 @@
       allocation_pool_end: "{{ os_subnet_range | ansible.utils.ipaddr('last_usable') }}"
       dns_nameservers: "{{ os_external_dns }}"
 
+  - name: 'Create the primary cluster IPv6 subnet'
+    openstack.cloud.subnet:
+      name: "{{ os_subnet6 }}"
+      network_name: "{{ os_network }}"
+      cidr: "{{ os_subnet6_range }}"
+      ip_version: 6
+      ipv6_address_mode: "slaac"
```

Comment:
I wonder if we should make this configurable by letting the user define it in the inventory instead given it's possible to set stateful, stateless and slaac. Thoughts?

Reply:
Yup! that can be easily done.